src/Routes.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Detail from '../src/pages/Detail/Detail';
+import Main from '../src/pages/Main/Main';
```

Comment:
동일한 디렉토리 레벨에 있기 때문에, 경로 다음과 같이 접근해주시면 됩니다!
```suggestion
import Detail from './pages/Detail/Detail';
import Main from './pages/Main/Main';
```

Reply:
네 감사합니다! 수정했습니다.